### PR TITLE
updated waitlist

### PR DESCRIPTION
**Pull request by ainobs**
Screenshot (28)

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -61,6 +61,40 @@ nav a:hover{
...
+    width: 30%;
+    display: grid;
+    border: 3px solid gray;
+    border-radius: 15px;
+    margin: 50px auto;
+    height: 480px;
+    background-color: white;
+    padding: 15px;
+}
+.test {
```
Comment (goke-epapa) [inline style.css]: can you give the class a more descriptive name? `test` seems like something you were trying out and not the final one.

Comment (ainobs) [reply style.css]: i was just using random class names that entered my head. i will work on it now

**Review by goke-epapa — APPROVED** [review]

**Comment by ainobs** [comment]
Screenshot (29)

**Comment by ainobs** [comment]
Screenshot (31)

**Review by faithakinbobola — APPROVED** [review]

**Review by Iyiola-stack — APPROVED** [review]

**Pull request merged**